README.md

```diff
@@ -0,0 +1,44 @@
+[![Codecov](https://img.shields.io/codecov/c/github/mariosimao/notion-sdk)](https://app.codecov.io/gh/mariosimao/notion-sdk)
+[![Type coverage](https://shepherd.dev/github/mariosimao/notion-sdk/coverage.svg)](https://shepherd.dev/github/mariosimao/notion-sdk)
+[![Notion API version](https://img.shields.io/badge/API%20Version-2021--08--16-blue)](https://developers.notion.com/reference/versioning)
+
+# notion-sdk-php
+
+A complete Notion SDK for PHP developers.
+## Installation
```

Comment:
Somewhere in this documentation, it is worth explaining that either `vimeo/psalm` or `phpstan/phpstan` should be used in combination with this library, to avoid silly/simple mistakes